SwanProjects/swanprojects/config.py

```diff
@@ -0,0 +1,33 @@
+from traitlets import Unicode
+from traitlets.config import Configurable
+import pathlib
+import os
+
+class SwanProjectsConfig(Configurable):
```

Comment:
Please add documentation to this. There are some variables that are not clear what they are.